space/clientspace/space.go

```diff
@@ -248,6 +255,51 @@ func (s *space) InstallBundledObjects(ctx context.Context) error {
 	return nil
 }
 
+func (s *space) migrationProfileObject(ctx context.Context) error {
```

Comment:
maybe move this as function with `clientspace.Space` parameter to `profilemigration` package?

Reply:
I tried initially, but it's really hard because of cyclic deps. even with interfaces